src/coroutine.rs

```diff
@@ -315,7 +344,9 @@ impl Drop for Handle {
         if state == State::Finished {
             ctx.resume(0);
         } else {
-            ctx.resume_ontop(self.0 as *mut _ as usize, coroutine_unwind);
+            self.unwind_flag.store(true, ::std::sync::atomic::Ordering::Release);
+            //ctx.resume_ontop(self.0 as *mut _ as usize, coroutine_unwind);
+            ctx.resume(self.0 as *mut _ as usize);
```

Comment:
Right here you could for instance resume the execution of the coroutine with a custom value like `usize::max_value()`